CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
     ``AutoCompleter`` which has since developed a dependency on ``cmd2`` methods. 
     * Removed ability to call commands in ``pyscript`` as if they were functions (e.g ``app.help()``) in favor
     of only supporting one ``pyscript`` interface. This simplifies future maintenance.
+    * No longer supporting C-style comments. Hash (#) is the only valid comment marker.
```

Comment:
@kotfu I'd like to make sure we get your input on these two potentially breaking changes.  I think they are good chances in terms of simplifying things and making the code more maintainable.  But I want to make sure this doesn't negatively impact anything you are working on.